oprf/oprf.go

```diff
@@ -0,0 +1,229 @@
+package oprf
```

Comment:
Make sure to include a brief documentation of the package.

Reply:
Added ;)